File: informatika_ismeretek/kozep/2019_okt/java/Bejegyzes.java
public class Bejegyzes{
    
    public final int ora;
    public final int perc;
    public final int adasok;
    public final String nev;
    
    public Bejegyzes(String line) {
        var split = line.split(";");
        
        ora = Integer.parseInt(split[0]);
        perc = Integer.parseInt(split[1]);
        adasok = Integer.parseInt(split[2]);
        nev = split[3];
    }
}

File: informatika_ismeretek/kozep/2019_okt/c#/Bejegyzes.cs
public class Bejegyzes {

    public readonly int ora;
    public readonly int perc;
    public readonly int adasok;
    public readonly string nev;

    public Bejegyzes(string line) {
        var split = line.Split(';');

        ora = int.Parse(split[0]);
        perc = int.Parse(split[1]);
        adasok = int.Parse(split[2]);
        nev = split[3];
    }
}